tests/string_utils_int/block_storage_string_utils_int.c

```diff
@@ -0,0 +1,272 @@
+// Copyright(C) Microsoft Corporation.All rights reserved.
+
+#ifdef __cplusplus
+#include <cinttypes>
+#include <clocale>
+#else
+#include <inttypes.h>
+#include <stdbool.h>
+#include <locale.h>
+#endif
+
+#include "windows.h"
+
+#include "testrunnerswitcher.h"
+
+#include "block_storage_string_utils.h"
+
+
+BEGIN_TEST_SUITE(block_storage_string_utils_int_tests)
```

Comment:
>block_storage_ [](start = 17, length = 14)

Seems to be string_utils

Reply:
Changed!

---
In reply to: [281848300](https://github.com/Azure/azure-c-shared-utility/pull/309#discussion_r281848300) [](ancestors = 281848300)